<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QDlgReport</class>
 <widget class="QDialog" name="QDlgReport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>698</width>
    <height>637</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>647</width>
    <height>637</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>车辆进入报表查询</string>
  </property>
  <widget class="QWebView" name="webView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>681</width>
     <height>590</height>
    </rect>
   </property>
   <property name="url">
    <url>
     <string>about:blank</string>
    </url>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>8</y>
     <width>48</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>截止时间</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btQuerySum">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>8</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>查询总计</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>8</y>
     <width>24</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>类别</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>17</x>
     <y>8</y>
     <width>48</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>起始时间</string>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dtEnd">
   <property name="geometry">
    <rect>
     <x>254</x>
     <y>9</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cbxType">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>9</y>
     <width>74</width>
     <height>20</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>全部</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>按年</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>按季</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>按月</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>按年周</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>按年月周</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>按日</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>按时</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>按省</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>按省内</string>
    </property>
   </item>
  </widget>
  <widget class="QDateTimeEdit" name="dtStart">
   <property name="geometry">
    <rect>
     <x>71</x>
     <y>9</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btPrint">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>8</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>打印</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btQueryDetail">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>8</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>查询明细</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKitWidgets/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
